### PR TITLE
OpenStack: tolerate incorrect urls with double / in the path

### PR DESCRIPTION
**Pull request by Fedosin**
In OSP13 there is a bug, that generates the url path with double /, i.e. http://10.1.52.2:5000//v3 in clouds.yaml file.

Keystone accepts this URL, but CIRO fails to parse it. Despite the fact this bug was fixed, some customers still have it. So, we have to be tolerant for this and support such a format as well.

### CODE REVIEW

File: pkg/storage/swift/swift.go [hunk 1]
```diff
@@ -267,6 +267,13 @@ func (d *driver) ensureAuthURLHasAPIVersion() error {
 
 	path := parsedURL.Path
 
+	// There is a bug in OSP13 that generates the url path with double /
+	// i.e. http://10.1.52.2:5000//v3
+	// We have to be tolerant for this and support such a format as well.
+	if strings.HasPrefix(path, "//") {
```
Comment (dmage) [inline pkg/storage/swift/swift.go]: What if someone generates the URL path with triple ///, `http://10.1.52.2:5000///v3`? Or uses `http://10.1.52.2:5000/./v3`?

Comment (Fedosin) [reply pkg/storage/swift/swift.go]: as you wish :)

**Pull request merged**